math/src/tests/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/calculator';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] It is great to have a snapshot test for the Calculator component. Similarly, make a simple snapshot test for each of the other components. :pray: 
- [ ] In addition, kindly add tests to simulate user interaction at least in the `MathContainer` component, using React Testing Library.